pkg/ui/src/views/reports/containers/range/index.tsx

```diff
@@ -82,38 +89,44 @@ class Range extends React.Component<RangeProps, {}> {
 
     // A bunch of quick error cases.
     if (!_.isNil(this.props.lastError)) {
-      return <ErrorPage
-        rangeID={rangeID}
-        errorText={`Error loading range ${this.props.lastError}`}
-      />;
+      return (
+        <ErrorPage
+          rangeID={rangeID}
+          errorText={`Error loading range ${this.props.lastError}`}
+        />
+      );
     }
     if (_.isEmpty(range)) {
-      return <ErrorPage
-        rangeID={rangeID}
-        errorText={`Loading cluster status...`}
-      />;
+      return (
+        <ErrorPage rangeID={rangeID} errorText={`Loading cluster status...`} />
+      );
     }
     const responseRangeID = FixLong(range.range_id);
     if (!responseRangeID.eq(rangeID)) {
-      return <ErrorPage
-        rangeID={rangeID}
-        errorText={`Updating cluster status...`}
-      />;
+      return (
+        <ErrorPage rangeID={rangeID} errorText={`Updating cluster status...`} />
+      );
     }
     if (responseRangeID.isNegative() || responseRangeID.isZero()) {
-      return <ErrorPage
-        rangeID={rangeID}
-        errorText={`Range ID must be a positive non-zero integer. "${rangeID}"`}
-      />;
+      return (
+        <ErrorPage
+          rangeID={rangeID}
+          errorText={`Range ID must be a positive non-zero integer. "${rangeID}"`}
+        />
+      );
     }
 
     // Did we get any responses?
     if (!_.some(range.responses_by_node_id, resp => resp.infos.length > 0)) {
-      return <ErrorPage
-        rangeID={rangeID}
-        errorText={`No results found, perhaps r${this.props.params[rangeIDAttr]} doesn't exist.`}
-        rangeResponse={range}
-      />;
+      return (
+        <ErrorPage
+          rangeID={rangeID}
+          errorText={`No results found, perhaps r${
+            this.props.params[rangeIDAttr]
+            } doesn't exist.`}
+          rangeResponse={range}
```

Comment:
kind of prefer the old wrapping here… maybe we should just go all in on prettier and not keep changing it's output though… idk